packages/react-aria-components/src/Disclosure.tsx

```diff
@@ -13,13 +13,66 @@
 import {AriaDisclosureProps, useDisclosure} from '@react-aria/disclosure';
 import {ButtonContext} from './Button';
 import {ContextValue, DEFAULT_SLOT, Provider, RenderProps, SlotProps, useContextProps, useRenderProps} from './utils';
-import {DisclosureState, useDisclosureState} from '@react-stately/disclosure';
-import {forwardRefType} from '@react-types/shared';
+import {DisclosureGroupState, DisclosureState, DisclosureGroupProps as StatelyDisclosureGroupProps, useDisclosureGroupState, useDisclosureState} from '@react-stately/disclosure';
+import {DOMProps, forwardRefType, Key} from '@react-types/shared';
+import {filterDOMProps, mergeProps, mergeRefs, useId} from '@react-aria/utils';
 import {HoverEvents, useFocusRing} from 'react-aria';
-import {mergeProps, mergeRefs} from '@react-aria/utils';
 import React, {createContext, DOMAttributes, ForwardedRef, forwardRef, ReactNode, useContext} from 'react';
 
-export interface DisclosureProps extends Omit<AriaDisclosureProps, 'children'>, HoverEvents, RenderProps<DisclosureRenderProps>, SlotProps {}
+export interface DisclosureGroupProps extends StatelyDisclosureGroupProps, RenderProps<DisclosureGroupRenderProps>, DOMProps {}
```

Comment:
Do we want `AriaLabelingProps` here?

Reply:
Maybe? DisclosureGroup doesn't have a role though, so not sure if that's allowed.